### PR TITLE
Chinese pages: remove space in More info link

### PR DESCRIPTION
**Pull request by marchersimon**
There should be no space between full-width punctuation and any characters. That also holds true for the `More information` link. I also updated the `set-more-info-link.py` script to leave out the space for `zh` and `zh_TW` pages.

### CODE REVIEW

File: pages.zh/common/vim.md [hunk 1]
```diff
@@ -23,16 +23,16 @@
 
 - 在文件中搜寻（按下 `n`/`N` 来在上一个 / 下一个结果中切换）：
 
-`<Esc> 更多信息： <Enter>`
+`<Esc> 更多信息：<Enter>`
 
 - 对整个文件使用正则表达式进行替换：
 
-`<Esc> 更多信息： <Enter>`
+`<Esc> 更多信息：<Enter>`
```
Comment (marchersimon) [inline pages.zh/common/vim.md:30]: I have no idea why this page is full of "More information" 🤔

Comment (blueskyson) [reply pages.zh/common/vim.md:30]: I used to review this page. I thought it means to ask users to see the information link. 😂

Comment (marchersimon) [reply pages.zh/common/vim.md:30]: Looks like I messed something up in #5240. I have no idea how, but with regex, anything is possible 😅

**Review by blueskyson — APPROVED** [review]

File: pages.zh/common/vim.md (hunk 1 above)
Comment (blueskyson) [reply pages.zh/common/vim.md:30]: I'll fix this page sometime.

Comment (marchersimon) [reply pages.zh/common/vim.md:30]: Thanks!

**Comment by blueskyson** [comment]
Just fix some conflicts.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**